src/components/menu-left/MenuLeft.vue

```diff
@@ -1,14 +1,68 @@
 <template>
   <div class="column is-narrow menu-left">
     <h3>Menu left</h3>
+    <section>
+      <PerSheetPreview
+        v-for="sheet in numberOfSheets"
+        :key="sheet + '-sheet'"
+        sheet="sheet"
+      />
+    </section>
+    <footer class="modal-card-foot">
+      <b-button
```

Comment:
We could consider using a number input from Buefy instead, which is basically the same thing: https://buefy.org/documentation/numberinput